android-sdk/src/main/java/com/microsoft/hsg/android/HealthVaultFileSettings.java

```diff
@@ -36,176 +39,179 @@
  */
 public class HealthVaultFileSettings implements HealthVaultSettings {
 
-    /** The properties. */
-    private Properties properties;
-    
-    /** The ctx. */
-    private Context ctx;
-    
-    /**
-     * Instantiates a new health vault file settings.
-     * 
-     * @param ctx the ctx
-     */
-    public HealthVaultFileSettings(Context ctx)
-    {
-        this.ctx = ctx;
-        
-        try
-        {
-            properties = new Properties();
-            try
-            {
-                properties.load(ctx.openFileInput("settings.props"));
-            }
-            catch(FileNotFoundException fnfe)
-            {
-            }
-        }
-        catch(IOException ioe)
-        {
-            throw new HVSystemException("Could not load properties.", ioe);
-        }
-    }
+	/** The properties. */
+	private Properties properties;
+
+	/** The ctx. */
+	private Context ctx;
+
+	/**
+	 * Instantiates a new health vault file settings.
+	 *
+	 * @param ctx the ctx
+	 */
+	public HealthVaultFileSettings(Context ctx) {
+		this.ctx = ctx;
+		FileInputStream inputStream = null;
+		try {
+			properties = new Properties();
+			try {
+				inputStream = ctx.openFileInput("settings.props");
```

Comment:
I've seen settings.props references other places in this PR. Perhaps it is best to centralize this value in a constant somewhere and referencing it where it is needed.